lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -2550,7 +2548,7 @@ def quiver(self, *args,
 
         **kwargs
             Any additional keyword arguments are delegated to
-            :class:`~matplotlib.collections.LineCollection`
+            :class:`~matplotlib.collections.Line3DCollection`
```

Comment:
Would need to be `mpl_toolkits.mplot3d.art3d.Line3DCollection`, but

```suggestion
            `.Line3DCollection`
```
is sufficient.